src/lib/components/Wave.svelte

```diff
@@ -8,7 +8,7 @@
 
 <style>
 	svg {
-		position: fixed;
+		position: absolute;
```

Comment:
what did this change?

Reply:
this fixes an issue where the wave may cause contrast issues when viewed in landscape on certain devices. For instance: 
![image](https://github.com/ReVanced/revanced-website/assets/56329333/175dc46c-690a-47bf-abd3-cb1c7a8a121c)